treeherder/config/settings.py

```diff
@@ -322,7 +325,23 @@ def server_supports_tls(url):
         'options': {
             'queue': 'fetch_bugs'
         }
-    }
+    },
+    'seta-analyze-failures': {
+        'task': 'seta-analyze-failures',
+        'schedule': timedelta(days=1),
+        'relative': True,
+        'options': {
+            'queue': "seta_analyze_failures"
+        }
+    },
+    'seta-load-preseed': {
+        'task': 'seta-load-preseed',
+        'schedule': timedelta(hours=2),
```

Comment:
preseed data doesn't change very often, only when we update it via a file.  I don't see a need to do this every 2 hours, maybe days=1 instead?